src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets

```diff
@@ -69,7 +69,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </PropertyGroup>
 
-  <Import Project="Microsoft.NET.Sdk.BlazorWebAssembly.ServiceWorkerAssetsManifest.targets" />
+  <Import Project="Microsoft.NET.Sdk.BlazorWebAssembly.ServiceWorkerAssetsManifest.targets" Condition="'$(ServiceWorkerAssetsManifest)' != ''" />
```

Comment:
Our test app always had a service worker. Importing this was causing issues in a project that did not have a service worker. I added build and publish tests for an app that's closer to our standalone template.

Reply:
Curious: what issues did it cause for projects that didn't have service workers?

Reply:
The targets unconditionally run when the file is imported: https://github.com/dotnet/aspnetcore/blob/master/src/Components/WebAssembly/Sdk/src/targets/Microsoft.NET.Sdk.BlazorWebAssembly.ServiceWorkerAssetsManifest.targets#L16. This results in spooky errors because it assumes a manifest file exists